<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Magic Wormhole</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="MessageTable" name="message_table">
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="error_label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 0, 0, 96);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Error Label</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="message_edit">
        <property name="maxLength">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="send_message_button">
        <property name="text">
         <string>Send Message</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="send_files_button">
        <property name="text">
         <string>Send Files...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="send_folder_button">
        <property name="text">
         <string>Send Folder...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageTable</class>
   <extends>QTableWidget</extends>
   <header>messagetable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
